tests/functions.sh

```diff
@@ -712,11 +712,11 @@ function sg_member_add {
     request $but_id $f
 }
 
-function set_nic_sg {
-    sg=$1
-    but_id=$2
-    nic_id=$3
-    echo "set nic $nic_id SG in butterfly $but_id"
+function sg_nic_set {
```

Comment:
You are not updating a security group, you are updating some nic parameters: better call this "nic_set_sg"